ndcube/tests/test_ndcube.py

```diff
@@ -910,3 +911,12 @@ def test_axis_world_coords_without_input(test_input, expected):
     for i in range(len(all_coords)):
         np.testing.assert_allclose(all_coords[i].value, expected[i].value)
         assert all_coords[i].unit == expected[i].unit
+
+
+@pytest.mark.parametrize("test_input,expected", [
+    (cubem.explode_along_axis(0), (2*u.pix, 3*u.pix, 4*u.pix)),
+    (cubem.explode_along_axis(1), (3*u.pix, 2*u.pix, 4*u.pix)),
+    (cubem.explode_along_axis(-2), (3*u.pix, 2*u.pix, 4*u.pix))
+])
+def test_explode_along_axis(test_input, expected):
```

Comment:
As part of this test, could you also check the type of the output, and also that the meta is ```None``` for the sub-cubes and the the sequence meta is the same as the original cube meta?

Reply:
Yes ! I was thinking that this test (taken from `ndcube_sequence`) is not relevant at all.

Reply:
Do you have an idea why `NDCube.explode_along_axis(axis=0)[0].meta` is an `OrderedDict()` ?
It should not be `None` ?

Where `NDCube.explode_along_axis(axis=0)` is a `NDCubeSequence` and `NDCube.explode_along_axis(axis=0)[0]` the first `NDCube` of the previous `NDCubeSequence`.